axonserver/src/main/java/io/axoniq/axonserver/localstorage/file/StorageProperties.java

```diff
@@ -115,6 +115,7 @@
             Duration.ofDays(7)
     };
     private String indexFormat;
+    private long messagesBufferLimit = -1L;
```

Comment:
The location of this property is not very intuitive, users would have to define a property with name axoniq.axonserver.storage.event.message-buffer-limit, which does not make sense as it has nothing to do with event storage.

Reply:
removed it from storage, Im reading meta directly from DB now